funcx_endpoint/funcx_endpoint/strategies/simple.py

```diff
@@ -47,7 +47,13 @@ def strategize(self, *args, **kwargs):
 
     def _strategize(self, *args, **kwargs):
         task_breakdown = self.interchange.get_outstanding_breakdown()
-        logger.info(f"Task breakdown {task_breakdown}")
+        num_tasks = 0
```

Comment:
no need to sum here. We can log if there is any non-zero entry, which is a bit faster.